asab/library/service.py

```diff
@@ -234,9 +234,8 @@ async def read(self, path: str) -> typing.Optional[typing.IO]:
 
 		return None
 
-
 	@contextlib.asynccontextmanager
-	async def open(self, path: str):
+	async def open(self, path: str, return_version: bool = False):
```

Comment:
We need to find another way how to do that ... and again, it is likely specific to `asab-library`